solidus_signifyd.gemspec

```diff
@@ -28,15 +28,15 @@ Gem::Specification.new do |s|
   s.executables = s.files.grep(%r{^exe/}) { |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  solidus_version = ['>= 2.0', '< 3']
+  solidus_version = ['>= 2.0', '< 4']
   s.add_dependency 'solidus_api', solidus_version
   s.add_dependency 'solidus_core', solidus_version
   s.add_dependency 'solidus_backend', solidus_version
 
   s.add_dependency 'active_model_serializers', '~> 0.10.7'
   s.add_dependency 'devise'
   s.add_dependency 'signifyd', '~> 0.1.5'
-  s.add_dependency 'solidus_support', '~> 0.5'
+  s.add_dependency 'solidus_support', '~> 0.8.1'
```

Comment:
I'd just use `'~> 0.8`, otherwise, when we release `0.9` it won't update. I think we can safely assume that people updating will not update to 0.8.0 if 0.8.1 is available.

Reply:
updated, thank you for the catch!